thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/db/dao/AnomalyResultDAO.java

```diff
@@ -35,21 +39,39 @@
           + "AND ((r.startTimeUtc >= :startTimeUtc AND r.startTimeUtc <= :endTimeUtc) "
           + "OR (r.endTimeUtc >= :startTimeUtc AND r.endTimeUtc <= :endTimeUtc))";
 
-  private static final String FIND_BY_TIME_AND_FUNCTION_ID = "SELECT r FROM AnomalyResult r WHERE r.function.id = :functionId "
-      + "AND ((r.startTimeUtc >= :startTimeUtc AND r.startTimeUtc <= :endTimeUtc) "
-      + "OR (r.endTimeUtc >= :startTimeUtc AND r.endTimeUtc <= :endTimeUtc))";
+  private static final String FIND_BY_TIME_AND_FUNCTION_ID =
+      "SELECT r FROM AnomalyResult r WHERE r.function.id = :functionId "
+          + "AND ((r.startTimeUtc >= :startTimeUtc AND r.startTimeUtc <= :endTimeUtc) "
+          + "OR (r.endTimeUtc >= :startTimeUtc AND r.endTimeUtc <= :endTimeUtc))";
 
-  private static final String FIND_BY_COLLECTION_TIME_AND_FUNCTION_ID = "SELECT r FROM AnomalyResult r WHERE r.function.collection = :collection "
-      + "AND r.function.id = :functionId "
-      + "AND ((r.startTimeUtc >= :startTimeUtc AND r.startTimeUtc <= :endTimeUtc) "
-      + "OR (r.endTimeUtc >= :startTimeUtc AND r.endTimeUtc <= :endTimeUtc))";
+  private static final String FIND_BY_COLLECTION_TIME_AND_FUNCTION_ID =
+      "SELECT r FROM AnomalyResult r WHERE r.function.collection = :collection "
+          + "AND r.function.id = :functionId "
+          + "AND ((r.startTimeUtc >= :startTimeUtc AND r.startTimeUtc <= :endTimeUtc) "
+          + "OR (r.endTimeUtc >= :startTimeUtc AND r.endTimeUtc <= :endTimeUtc))";
 
   private static final String FIND_VALID_BY_TIME_EMAIL_ID =
       "SELECT r FROM EmailConfiguration d JOIN d.functions f JOIN f.anomalies r "
-      + "WHERE d.id = :emailId AND ((r.startTimeUtc >= :startTimeUtc AND r.startTimeUtc <= :endTimeUtc) "
-      + "OR (r.endTimeUtc >= :startTimeUtc AND r.endTimeUtc <= :endTimeUtc)) "
+          + "WHERE d.id = :emailId AND ((r.startTimeUtc >= :startTimeUtc AND r.startTimeUtc <= :endTimeUtc) "
+          + "OR (r.endTimeUtc >= :startTimeUtc AND r.endTimeUtc <= :endTimeUtc)) "
           + "AND r.dataMissing=:dataMissing";
 
+  private static final String COUNT_GROUP_BY_FUNCTION = "select count(r.id) as num, r.function.id,"
```

Comment:
I feel WHEN the WORDS jumbled IN both CAPS and LOWERCASE, it MAKES things HARDER to READ.
Using all lowercase and only only entities camel case is better to read. There is no difference performance wise. Also IDE highlights the sql keywords.

There has been many discussions around this http://stackoverflow.com/questions/608196/why-should-i-capitalize-my-sql-keywords